RecoTracker/IterativeTracking/python/LowPtTripletStep_cff.py

```diff
@@ -55,7 +57,8 @@
     originRadius = 0.02,
     nSigmaZ = 4.0
 ))
-trackingPhase1.toModify(lowPtTripletStepTrackingRegions, RegionPSet = dict(ptMin = 0.35)) # FIXME: Phase1PU70 value, let's see if we can lower it to Run2 value (0.2)
+trackingPhase1.toModify(lowPtTripletStepTrackingRegions, RegionPSet = dict(ptMin = 0.2))
```

Comment:
@slava77 you can see some tests that I ran a while ago (810pre12) with this configuration:
https://fpantale.web.cern.ch/fpantale/offline_tuning/final/plots_timing/iterations.pdf
Many things happened in the mean time, but you can get the order of magnitude of the effect on timings on iterations.